tools/run.py

```
@@ -76,12 +77,13 @@ def convert_bash_to_powershell(bash_command: str) -> str:
 
     def translate_path(path: str) -> str:
         """Translate Unix-style paths to Windows-style paths."""
+        PROJECT_DIR = get_constant('PROJECT_DIR')
```

Ruff (0.8.2)

Undefined name `get_constant`

(F821)